packages/contracts-bedrock/contracts/L1/L2OutputOracle.sol

```diff
@@ -204,59 +191,129 @@ contract L2OutputOracle is Initializable, Semver {
             // blockhash value, and delay submission until it is confident that the L1 block is
             // finalized.
             require(
-                blockhash(_l1BlockNumber) == _l1Blockhash,
+                blockhash(_l1BlockNumber) == _l1BlockHash,
                 "L2OutputOracle: blockhash does not match the hash at the expected height"
```

Comment:
Nit: The change of casing in the variable suggests that the error message should become
```suggestion
                "L2OutputOracle: block hash does not match the hash at the expected height"
```